ui/src/components/credentials/ScanLink.tsx

```diff
@@ -97,38 +105,47 @@ export function ScanCredentialLink() {
 
   const onStartAgain = () => {
     makeRequestAbortable(createCredentialQR);
-    setCredentialQRCheck({ status: CredentialQRStatus.Pending });
+    setCredentialQRCheck({ status: "pending" });
   };
 
-  if (shareCredentialQRCode.status === "failed") {
-    if (shareCredentialQRCode.error.status === HTTPStatusError.NotFound) {
+  const hasFailed = hasAsyncTaskFailed(shareCredentialQRCode)
+    ? shareCredentialQRCode.error
+    : hasAsyncTaskFailed(credentialQRCheck)
+    ? credentialQRCheck.error
+    : undefined;
+
+  if (hasFailed) {
+    if (hasFailed.status === HTTPStatusError.NotFound) {
       return (
-        <Space
-          align="center"
-          direction="vertical"
-          size="large"
-          style={{ textAlign: "center", width: 400 }}
-        >
-          <Avatar className="avatar-color-primary" icon={<QRIcon />} size={56} />
+        <Space align="center" direction="vertical" size="large">
+          <Avatar className="avatar-color-error" icon={<QRIcon />} size={56} />
 
-          <Typography.Title level={2}>Credential link is invalid</Typography.Title>
+          <Typography.Title level={2}>Claim link is invalid</Typography.Title>
```

Comment:
Why is this "claim" and not "credential"? Also a [question](https://www.figma.com/file/BE3tUbvtSB2EqmJsybpeFC?node-id=5792:165929#425519941) for Liudas.